recipes/detools/all/CMakeLists.txt

```diff
@@ -1,9 +1,20 @@
 cmake_minimum_required(VERSION 3.1)
```

Comment:
Can you bump this minimum required version to `3.15`?

Reply:
@jwillikers 
Thank you for your review!
I fixed these points.

Reply:
Thanks! LGTM